aaelf64/aaelf64.rst

```diff
@@ -1826,116 +1834,27 @@ At this time this ABI specifies no generic platform architecture compatibility d
 Program Property
 ----------------
 
-The following processor-specific program property types [LINUX_ABI_] are
-defined:
-
-.. table:: Program Property Type
-
-    +-----------------------------------------+------------+
-    | Name                                    | Value      |
-    +=========================================+============+
-    | GNU\_PROPERTY\_AARCH64\_FEATURE\_1\_AND | 0xc0000000 |
-    +-----------------------------------------+------------+
-
-``GNU_PROPERTY_AARCH64_FEATURE_1_AND`` describes a set of processor features
-with which an ELF object or executable image is compatible, but does not
-require in order to execute correctly.  It has a single 32-bit value for the
-``pr_data`` field.  Each bit represents a separate feature.
-
-Static linkers processing ELF relocatable objects must set the feature bit in
-the output object or image only if all the input objects have the corresponding
-feature bit set. For each feature bit set in an ELF executable or shared library,
-a loader may enable the corresponding processor feature for that ELF file.
-
-The following bits are defined for GNU_PROPERTY_AARCH64_FEATURE_1_AND:
-
-.. table:: GNU_PROPERTY_AARCH64_FEATURE_1_AND Bit Flags
-
-    +-----------------------------------------+------------+
-    | Name                                    | Value      |
-    +=========================================+============+
-    | GNU\_PROPERTY\_AARCH64\_FEATURE\_1\_BTI | 1U << 0    |
-    +-----------------------------------------+------------+
-    | GNU\_PROPERTY\_AARCH64\_FEATURE\_1\_PAC | 1U << 1    |
-    +-----------------------------------------+------------+
-
-``GNU_PROPERTY_AARCH64_FEATURE_1_BTI`` This indicates that all executable
-sections are compatible with Branch Target Identification mechanism. An
-executable or shared object with this bit set is required to generate
-`Custom PLTs`_ with BTI instruction.
-
-``GNU_PROPERTY_AARCH64_FEATURE_1_PAC`` This indicates that all
-executable sections have been protected with Return Address Signing.
-Its use is optional, meaning that an ELF file where this feature bit
-is unset can still have Return Address signing enabled in some or all of
-its executable sections.
+The information on Program Property has been moved to [SYSVABI64_].
```

Comment:
The information on Program Property has |been| moved to....

Repeat for the next 3 sections.